packages/react-ui-map/src/Mapbox/Mapbox.js

```diff
@@ -67,7 +67,11 @@ export default class Mapbox extends PureComponent {
 
   shouldComponentUpdate(nextProps, nextState) {
     return (nextProps.children !== this.props.children ||
-      nextState.map !== this.state.map)
+      nextState.map !== this.map)
```

Comment:
this condition, `nextState.map !== this.map` no longer makes sense. As `map` isn't part of state, and `this.map` should get destroyed on component unmount anyway